src/common/s3-asset.ts

```diff
@@ -88,6 +83,19 @@ export function uploadBuiltInJarsAndRemoteFiles(
   };
 }
 
+function getSourceAsset(props: BuildJarProps, bundling: BundlingOptions) {
+  if (process.env.CI === 'true') {
```

Comment:
You should change to `false`. We must do full validation in CI. You might ignore some code built in local development.